aspnetcore/security/authentication/certauth.md

```diff
@@ -206,3 +206,249 @@ Complete the following steps In IIS Manager:
 ### Azure and custom web proxies
 
 See the [host and deploy documentation](xref:host-and-deploy/proxy-load-balancer#certificate-forwarding) for how to configure the certificate forwarding middleware.
+
+### Using certificate authentication with the X-ARR-ClientCert header
+
+The AddCertificateForwarding method is used so that the client header can be specified and how the certificate is to be loaded using the HeaderConverter option. When sending the certificate with the HttpClient using the default settings, the ClientCertificate was always be null. The X-ARR-ClientCert header is used to pass the client certificate, and the cert is passed as a string to work around this.
```

Comment:
```suggestion
The `AddCertificateForwarding` method is used so that the client header can be specified and how the certificate is to be loaded using the `HeaderConverter` option. When sending the certificate with the `HttpClient` using the default settings, the `ClientCertificate` will always be null. The `X-ARR-ClientCert` header is used to pass the client certificate. The certificate is passed as a string to work around this.
```